### ISSUE TITLE
Question about IP Management: Is there an API available to assigning/removing additional IP

### ISSUE THREAD

**Issue by ar13pit** [contabo/cntb#33]
I'm unable to find any documentation on how to assign/remove additional IP address to an instance. This is possible via the Contabo Control Panel > IP Management. However, to automate the usage of `keepalived` with a Virtual IP address, it becomes necessary to be able to assign the additional public IP address to an instance at least programmatically, if assigning it to multiple instances is not supported.

Could you please comment/give any directions about this?

**Comment by tahergalal** [comment]
Yes we are working on providing an API that will be able to perform this but currently there is no api therefore this is not yet implemented. But We are working on it.

**Comment by Nialit** [comment]
confirm, much wanted feature
